docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md

```diff
@@ -9,14 +9,26 @@ Besu nodes can operate (that is, submit transactions and receive blocks) in a Qu
 
 To connect to a Quorum IBFT 1.0 network:
 
-1. In the Quorum IBFT 1.0 [genesis file](../Genesis-File.md), update the consensus protocol specified in the `config`
-   property from `istanbul` to `ibft`.
+1. In the [Quorum IBFT 1.0 genesis file](https://docs.goquorum.consensys.net/en/stable/HowTo/Configure/Consensus-Protocols/IBFT/#genesis-file):
+
+    - Update the consensus protocol specified in the `config` item from `istanbul` to `ibft`.
+    - In the `ibft` item:
+        - Change `epoch` to `epochlength`.
+        - Add `blockperiodseconds`, the minimum block time in seconds.
+        - Add `requesttimeoutseconds`, the timeout for each consensus round before a round change, in seconds.
+        - Set `policy` to 0.
```

Comment:
We shouldn't include the policy field isn't supported in Besu. Besu always rotates validators.

Reply:
Fixed.